packages/esm-ohri-pmtct-app/src/forms/mother-postnatal/1.0.json

```diff
@@ -14,7 +14,9 @@
                       "id":"ptrackerid",
                       "questionOptions":{
                          "concept":"6c45421e-2566-47cb-bbb3-07586fffbfe2",
-                         "rendering":"text"
+                         "rendering":"text",
+                         "max": "12",
```

Comment:
Have you tested this? I don't think it works for text.

Reply:
@eudson @kajambiya  [Response here](https://ohri.atlassian.net/wiki/spaces/HOME/pages/274890755/Business+rules+Mother+post+natal+form?focusedCommentId=277282818)